docs/blog/write-the-docs-2018-kickoff.rst

```diff
@@ -0,0 +1,86 @@
+.. post:: Jan 20, 2018
+   :tags: community updates, core team
+   :author: Mikey Ariel
+
+Write the Docs 2018 Kick-Off
+============================
+
+Greetings, global documentarians!
+
+Mikey here, writing from the deep, dark cold European winter, but looking forward to spring and summer, full of sunshine and docs!
+Following Eric's winter-warming `2017 Stats <http://www.writethedocs.org/blog/write-the-docs-2017-stats/>`_ post, I'd like to highlight some of the activities, changes, and experiments that you can expect in 2018.
+
+These efforts are a part of optimizing our work in supporting and growing the community, and the also aim to keep things fresh and exciting for our documentarians.
```

Comment:
"part of" not "a part of" ? missing "y" on they. But the two sentence halves read arkward to me. How about

> These efforts are part of our work to support and grow the community, and we hope they keep things fresh and exciting for documentarians.

Reply:
Although if we do that we should remove the next "hope" below :-p